src/main/java/net/io/kino/controller/LogsController.java

```diff
@@ -0,0 +1,44 @@
+package net.io.kino.controller;
+
+import net.io.kino.model.loggingaction.EventData;
+import net.io.kino.model.loggingaction.EventType;
+import net.io.kino.service.logger.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequestMapping("/logs")
+public class LogsController {
+
+    Logger logger;
+
+    private LogsController(@Autowired Logger logger) {
+        this.logger = logger;
+    }
+
+    @GetMapping("/")
```

Comment:
Zmień `@GetMapping("/")` na po prostu `@GetMapping` bo jak masz `/` to wymusza to używanie adresu
`http://localhost:8080/logs/`

bez tego można albo
`http://localhost:8080/logs/`
albo
`http://localhost:8080/logs`